sources/packages/backend/apps/db-migrations/src/sql/Announcements/Create-announcements-table.sql

```diff
@@ -0,0 +1,34 @@
+CREATE TABLE IF NOT EXISTS sims.announcements (
+    id SERIAL PRIMARY KEY,
+    message_title VARCHAR(100) NOT NULL,
+    message VARCHAR(200) NOT NULL,
+    start_date TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW (),
+    end_date TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW (),
+    target VARCHAR(200) NOT NULL,
+    -- Audit columns
+    created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+    updated_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+    creator INT NULL DEFAULT NULL REFERENCES sims.users(id) ON DELETE
+    SET
+        NULL,
+        modifier INT NULL DEFAULT NULL REFERENCES sims.users(id) ON DELETE
+    SET
+        NULL
+);
```

Comment:
Audit columns are not required to this table as it is not going to be updated through the system.

Reply:
If audit columns are removed, it can not be saved with the .save function in e2e tests. I originally did not have them.

Reply:
When audit columns, apparently we should not be extending the `RecordDataModel` as the `RecordDataModel` and it's child class `BaseModel` are class representing audit columns. 
![image](https://github.com/user-attachments/assets/ced289e9-9f29-401b-9bd8-19d1e83d270f)

Removing the extends will let us to remove the audit columns from table.

Reply:
removed